### ISSUE TITLE
Always use highest resolution textures/models

### ISSUE THREAD

**Issue by gtype** [Kaldaien/UnX#33]
Just like the title says, make the game always use the highest resolution textures/models

**Comment by Kaldaien** [comment]
I can do the first (textures), but not models.

Finding the hashes for the high and low-res version of each texture would be a lengthy process, but texture modders appear to be doing a lot of the work for me  over time ;)

**Comment by gtype** [comment]
Sure, if you just add some way for it to be possible, I'm sure the community will step up to it. Can I ask why you can't do models? Doesn't Directx have some sort of hook on some loadmodel function or something? :P
Thanks

**Comment by Kaldaien** [comment]
No, models are much lower-level. There's no DirectX format for models. Texture data is just an array of pixels, no interpretation of a data structure is necessary to manipulate them.

Models, on the other hand, have custom vertex attributes, various weird data alignment rules, proprietary skinning and model rigging for skeletal animation, etc. There's no way this will ever happen.
